rollup.config.js

```diff
@@ -68,11 +69,15 @@ const config = outputs.map(({ file, format }) => ({
       inline: false,
       plugins: postcssPlugins,
     }),
+    ts({
+      transpiler: 'babel',
+      tsconfig: (resolvedConfig) => ({ ...resolvedConfig, noEmit: false }),
```

Comment:
There's a bug in the react-testing script that it gets angry when we change noEmit in the tsconfig file directly

Reply:
This should be a comment in the code, too. 😉

Reply:
done!